service/src/main/java/uk/gov/hmcts/reform/fpl/service/TaskListRenderer.java

```diff
@@ -0,0 +1,172 @@
+
+        final TaskSection applicationDetails = newSection("Add application details", of(
+                tasks.get(CASE_NAME),
+                tasks.get(ORDERS_NEEDED),
+                tasks.get(HEARING_NEEDED)
+        ));
+
+        final TaskSection applicationGrounds = newSection("Add grounds for the application", of(
+                tasks.get(GROUNDS),
+                tasks.get(RISK_AND_HARM)
+                        .withHint("In emergency cases, you can send your application without this information"),
+                tasks.get(FACTORS_AFFECTING_PARENTING)
+                        .withHint("In emergency cases, you can send your application without this information")
+        ));
+
+        final TaskSection documents = newSection("Add supporting documents", of(tasks.get(DOCUMENTS)))
+                .withHint("For example, the social work chronology and care plan");
+
+        final TaskSection parties = newSection("Add information about the parties",
+                List.of(
+                        tasks.get(APPLICANT),
+                        tasks.get(CHILDREN),
+                        tasks.get(RESPONDENTS)
+                ));
+
+        final TaskSection courtRequirements = newSection("Add court requirements", of(
+                tasks.get(ALLOCATION_PROPOSAL)
+        ));
+
+        final TaskSection additionalInformation = newSection("Add additional information", of(
+                tasks.get(OTHER_PROCEEDINGS),
+                tasks.get(INTERNATIONAL_ELEMENT),
+                tasks.get(OTHERS),
+                tasks.get(ATTENDING_THE_HEARING)
+        )).withInfo("Only complete if relevant");
+
+        final TaskSection sentApplication = newSection("Send application", of(tasks.get(SUBMIT_APPLICATION)));
+
+        lines.add("<div class='width-50'>");
+
+        Stream.of(
+                applicationDetails,
+                applicationGrounds,
+                documents,
+                parties,
+                courtRequirements,
+                additionalInformation,
+                sentApplication).forEach(section -> lines.addAll(renderSection(section)));
+
+        lines.add("</div>");
+
+        return String.join("\n\n", lines);
+    }
+
+    private List<String> renderSection(TaskSection sec) {
+        final List<String> section = new LinkedList<>();
+
+        section.add(renderNewLine());
+        section.add(renderHeader(sec.getName()));
+
+        sec.getHint().map(this::renderHint).ifPresent(section::add);
+        sec.getInfo().map(this::renderInfo).ifPresent(section::add);
+
+        section.add(renderHorizontalLine());
+        sec.getTasks().forEach(task -> {
+            section.addAll(renderTask(task));
+            section.add(renderHorizontalLine());
+        });
+
+        return section;
+    }
+
+    private List<String> renderTask(Task task) {
+        final List<String> lines = new LinkedList<>();
+        if (task.getState() == NOT_AVAILABLE) {
```

Comment:
Do you think a switch might be nicer and more expanable here?

Reply:
done, in the end case is a bit longer but a bit more flatter